Update SRG-OS-000114-GPOS-00059 for RHEL 9 STIG

jan-cerny opened the pull request:
#### Description:

adds fix texts, Ansible remediations, fixes OCIL

#### Rationale:

RHEL 9 STIG

Mab879 reviewed (commented):
Just a few comments. If you can provide a source for device IDs that would be helpful.

Mab879 commented on linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml:
```diff
@@ -56,3 +57,21 @@ ocil: |-
+fix: |-
+    Configure GNOME 3 to disable automated mount of removable media.
+
+    First, add or update the following section
```
That sounds reasonable.

Mab879 commented:
/retest

jan-cerny commented:
/retest

jan-cerny commented:
/retest

jan-cerny commented:
I have rebased this pull request on the top of the currrent master branch and I have replaced fix by fixtext and resolved conflict with moving Jinja macro to a different file.

jan-cerny commented:
/retest

Mab879 commented on linux_os/guide/system/permissions/mounting/service_autofs_disabled/rule.yml:
```diff
@@ -60,9 +60,15 @@ references:
         servicename: autofs
+
+fixtext: |-
+    Configure the operating system to disable the ability to automount devices.
```
```suggestion
    Configure {{{ full_name }}} to disable the ability to automount devices.
```

Mab879 reviewed (changes requested):
Just one minor change requested.

Thanks for the PR.

Mab879 commented:
/retest

Mab879 commented:
@jan-cerny Can you take look a the merge conflict?

jan-cerny commented:
/retest

jan-cerny commented:
Hi, it's now rebased again.

Mab879 commented:
/retest

Mab879 commented:
/retest

Mab879 commented:
So those OpenShift CI errors are giving me some pause before I merged them.

@rhmdnd or @Vincent056 I think these failures might be valid, can you please confirm?

jan-cerny commented:
I have resolved the conflict and I have rebased on the top of the latest upstream branch.

Mab879 commented:
/retest

Vincent056 commented:
Since https://github.com/ComplianceAsCode/content/pull/8564 has just merged, the test should pass once this PR rebase.

jan-cerny commented:
I have rebased on the top of the latest upstream branch.

Mab879 commented:
/retest

Mab879 commented:
Waving SSGTS, the tests pass locally.
